src/main/java/com/employeesService/EmployeesService/exception/EmployeeExceptionHandler.java

```diff
@@ -0,0 +1,19 @@
+package com.employeesService.EmployeesService.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class EmployeeExceptionHandler {
```

Comment:
There should be one global exception handler, name it something like `GlobalExceptionHandler` and then have the annotated methods with `@ExceptionHandler` there both for Department and Employee exceptions